src/RelatedViewManager.js

```diff
@@ -35,17 +37,23 @@ define('Sage/Platform/Mobile/RelatedViewManager',  [
     domConstruct,
     query,
     array,
-    SDataStore
+    SDataStore,
+    RelatedViewWidget,
+    RelatedViewDetailWidget
 ) {
+    var _widgetTypes = {};
     return declare('Sage.Platform.Mobile.RelatedViewManager', null, {
 
-        id: 'relatedView',
+        id: 'relatedViewManager',
         relatedViews: null,
         relatedViewConfig: null,
+        widgetTypes: _widgetTypes,
```

Comment:
I'm pretty sure object refs in declare become static, so you probably don't need the _widgetTypes defined above here, just do widgetTypes: {}